UI/Pages/Cookies.cshtml

```diff
@@ -1,10 +1,134 @@
+            <li>remember the settings you’ve chosen</li>
+        </ul>
+        <p class="govuk-body">
+            Cookies on ‘Find a tuition partner’ are not used for advertising or marketing.
+        </p>
+        <h1 class="govuk-heading">Analytics cookies to help improve this service</h1>
+
+        <p class="govuk-body">
+            If you give permission, we use Google Analytics to measure how you use this service. This means we can update and improve it more effectively. We do not allow Google to use or share the data about how you use it.
+        </p>
+
+        <p class="govuk-body">Google Analytics stores information about:</p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>the pages you visit in this service</li>
+            <li>how long you spend on each page</li>
+            <li>how you got to the service</li>
+            <li>what you click on while you’re using the servic</li>
+        </ul>
+
+        <p class="govuk-body">Google Analytics does not store:</p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>any of your personal details, for example, your name</li>
+            <li>what you enter into pages on this site</li>
+        </ul>
+
+        <form method="post">
+            <div class="govuk-form-group">
+                <fieldset class="govuk-fieldset">
+                    <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+                        <p>
+                            Do you agree to the Department for Education using Google Analytics to help improve the Find a tuition partner service?
+                        </p>
+                    </legend>
+                    <div class="govuk-radios">
+                        <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" data-testid="cookie-consent-accept" id="cookie-consent-accept" name="consent" type="radio" value="true" checked="@Model.Consent">
+                            <label class="govuk-label govuk-radios__label" for="cookie-consent-accept">
+                                Yes, opt-in to analytics cookies
+                            </label>
+                        </div>
+                        <div class="govuk-radios__item">
+                            <input class="govuk-radios__input" data-testid="cookie-consent-deny" id="cookie-consent-deny" name="consent" type="radio" value="false" checked="@(Model.Consent == false)">
+                            <label class="govuk-label govuk-radios__label" for="cookie-consent-deny">
+                                No, do not use cookies to track my website usage
+                            </label>
+                        </div>
+                    </div>
+                </fieldset>
+            </div>
+            <input type="submit" data-testid="save-changes" value="Save changes" class="govuk-button" data-module="govuk-button">
```

Comment:
The [content doc](https://docs.google.com/document/d/1UUkpfkyhyg12aQDF17--H0eMRYB3IlMmmh7hT3QA2SI/edit#heading=h.ns60qtm90eqj) appears to want the button to be labelled "Save my choice"

![image](https://user-images.githubusercontent.com/201727/182298289-b0f66530-5822-4b4a-8e0f-920f7c94df9e.png)

Reply:
Changed